### PR TITLE
don’t init notifications for admins or mentors

### PR DESCRIPTION
**Pull request by learykara**

### CODE REVIEW

File: src/AppBar/notifications/NotificationActions.jsx [hunk 1]
```diff
@@ -24,20 +24,21 @@ const shouldInitNotifications = () => {
     CONFIG &&
     USER &&
     _.get(CONFIG, 'vendor.getstream.userFeedToken') &&
-    USER.access.indexOf(DESIGN_SYS_FLAG) === -1
-  )
-}
+    USER.access.indexOf(DESIGN_SYS_FLAG) === -1 &&
+    !/admin|mentor/.test(user.role)
```
Comment (ripleyaffect) [inline src/AppBar/notifications/NotificationActions.jsx]: `USER`?

Comment (learykara) [reply src/AppBar/notifications/NotificationActions.jsx]: gawd damnit.

Comment (diezguerra) [reply src/AppBar/notifications/NotificationActions.jsx]: oh damn

Comment (diezguerra) [reply src/AppBar/notifications/NotificationActions.jsx]: that's what you get for asking amateurs to review 😅

Comment (learykara) [reply src/AppBar/notifications/NotificationActions.jsx]: and letting amateurs write code 🤦‍♀️

**Pull request merged**